auto_nag/utils.py

```diff
@@ -619,6 +619,32 @@ def nice_round(val):
     return int(round(100 * val))
 
 
+def is_bot_email(email: str) -> bool:
+    """Check the email is belong to a bot account.
+
+    Args:
+        email: the account login email.
+    """
+    return email.endswith(".bugs") or email.endswith("@mozilla.tld") or "-bot@" in email
```

Comment:
What is an example of a bot that matches `"-bot@" in email` but doesn't match the other two conditions?

Reply:
`phab-bot@bmo.tld` is an example. 

Do you think `.endswith(".tld")` could replace `.endswith("@mozilla.tld")` and `"-bot@" in email`?

Reply:
Not sure, but I guess .tld is not really usable so makes sense

Reply:
I will replace `.endswith("@mozilla.tld")` with `.endswith(".tld")`. Do you prefer dropping `"-bot@" in email`?

Reply:
Yeah, there could be legit people with "-bot@" in their email address (though unlikely...).